docs/content/media/s3.md

```diff
@@ -0,0 +1,17 @@
+## S3
+
+You can use [Amazon S3](https://aws.amazon.com/s3/) or any other S3 compatible storage to handle your image uploads in HedgeDoc.
```

Comment:
```suggestion
You can use [Amazon S3](https://aws.amazon.com/s3/) or any other S3-compatible storage (like [MinIO](https://min.io) or [Ceph Object Gateway](https://docs.ceph.com/en/latest/radosgw/)) to handle your image uploads in HedgeDoc.
```

We should probably mention some alternatives here and provide an example config?

Reply:
Not sure what example config we should provide here? I added the suggested change.